clang/lib/CIR/CodeGen/CIRGenFunction.h

```diff
@@ -509,6 +509,36 @@ class CIRGenFunction : public CIRGenTypeCache {
 public:
   Address createTempAlloca(mlir::Type ty, CharUnits align, mlir::Location loc,
                            const Twine &name, bool insertIntoFnEntryBlock);
+
+  //===--------------------------------------------------------------------===//
+  //                         OpenACC Emission
+  //===--------------------------------------------------------------------===//
+public:
+  mlir::LogicalResult
+  emitOpenACCComputeConstruct(const OpenACCComputeConstruct &S);
```

Comment:
```suggestion
  emitOpenACCComputeConstruct(const OpenACCComputeConstruct &s);
```
MLIR naming style throughout the changes to this header

Reply:
Ooof... I was doing so well :D  I caught myself on this one a dozen times so far, I'm guessing beating the old coding standard out of my fingers is going to take a while :)